### DISCUSSION TITLE
Move yo teams NPM to officedevpnp account

### DISCUSSION THREAD

**Discussion by stephanbisser** [pnp/generator-teams#329] [General]
We should move the npm packages for yo teams from @wictorwilen's account maybe? So that it aligns with the rest of the PnP packages?

**Comment by RickVanRousselt** [comment]
Todo (First draft)

- Merge (or decline) all existing pull requests into next branch
- Release into Main branch (rename branch)
- Remove preview package from NPM
- Remove preview branch from GitHub
- Delete all preview packages (except the last) from NPM
- Allow OfficeDev npm account full control on NPM teams generator packages
- Add contributors to NPM teams generator packages
- Get M365 CLI Script to remove old versions in NPM (see M365 repo)

**Reply by wictorwilen** [reply]
`officedevpnp` is added as an owner to the packages

**Reply by RickVanRousselt** [reply]
@wictorwilen Can you do it again? We just missed the deadline to accept with a few days ;-) Thx

**Reply by RickVanRousselt** [reply]
On second thought @wictorwilen , can you also invite Stephan and me (rickvanrousselt &  stephanbisser) then we can clean up the old packages and the docs.

**Reply by wictorwilen** [reply]
Done

**Comment by stephanbisser** [comment]
- [x] Merge (or decline) all existing pull requests into preview branch
- [x] Release into Main branch (rename branch)
- [x] Remove next package from NPM
- [x] Remove next branch from GitHub
- [x] Delete all next packages (except the last) from NPM
- [x] Allow OfficeDev npm account full control on NPM teams generator packages
- [x] Add contributors to NPM teams generator packages
- [ ] Get M365 CLI Script to remove old versions in NPM (see M365 repo)
